### ISSUE TITLE
Is this repo still alive?

### ISSUE THREAD

**Issue by gonzarascon** [malerba118/react-sandbox-editor#10]
Hi!
I ran into this repo by accident and instantly loved it!
 I'm using the component on a personal project and I was wondering if it is still having support!

**Comment by malerba118** [comment]
I'm glad you've gotten some use out of it! I'm not actively working on it, but i'm also not opposed to doing some minor releases if there's demand for it. Do you have a specific bug or feature in mind?

**Comment by gonzarascon** [comment]
Yes! I was thinking on adding a console tab, so if the `scriptEditor` has any `console.log` (or warn, info, etc) it can be outputted there.

**Comment by malerba118** [comment]
Ahh yes, yes, I considered adding console options back in the day. To clarify, were you thinking about implementing this functionality yourself and making a PR against this repo or is this something you'd like me to add?

**Comment by gonzarascon** [comment]
Well, I was wondering if you could do that. I think that implementing that feature could take me a while and I'm not completely sure how to do it.

**Comment by malerba118** [comment]
Sure thing, makes sense. Is there a time you need this by? If not, I'll try to have something out in a week or two.

**Comment by gonzarascon** [comment]
There's no hurry, take your time! I think that this component has a lot to offer! Good work! :)
